SU-SWS/stanford_paragraph_types
# Earth 375 video hero

# READY FOR REVIEW

# Summary
- Added fields to video type for overlay
- Changed video type to use hero banner pattern
- Changed hero banner type to use responsive image formatter

# Needed By (Date)
- End of Sprint

# Urgency
- Medium (they really want it)

# Steps to Test

1. Checkout this branch and same branch in [stanford_components](https://github.com/SU-SWS/stanford_components/pull/87)
2. clear cache & `features-import stanford_paragraph_video stanford_paragraph_hero_banner -y`
3. Review home page hero banner and any other hero banners look normal still and that it contains a `<picture>` tag now instead of background image
4. Create complex page with video type
5. Set the display variant to full width
6. verify the video is playable and the text overlays with a hairline (maybe a style change will change this for non-full width display? )

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_components/pull/87

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
